tools/rtpsrelay/DomainStatisticsListener.cpp

```diff
@@ -0,0 +1,51 @@
+#include "DomainStatisticsListener.h"
+
+#include "utility.h"
+
+#include "lib/RelayTypeSupportImpl.h"
+#include "lib/QosIndex.h"
+
```

Comment:
```suggestion

#include <iostream>
```

attempt to fix some CI builds